.github/workflows/spectral-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Spectral Lint Commited OpenAPI Spec
+
+# Trigger the workflow on pull requests and pushes to the main branch
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
could you add [path filter](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) to run this flow only if the PR is changes the file in openapi folders + github workflow files + spectral.yaml?

Reply:
thanks added!